main.py

```diff
@@ -152,14 +152,14 @@ def save_passwords():
 
 
 # Function to load passwords from a JSON file 
-def load_passwords(file_path="vault.json"):
+def load_passwords(file_path="vault.txt"):
      try:
         with open(file_path, 'r') as file:
             data = json.load(file)
-            return data
-    except FileNotFoundError:
-        print(f"File not found: {file_path}")
-        return None
```

Comment:
Keep the exception, just not the return
`except FileNotFoundError:
        print(f"File not found: {file_path}")`